### PR TITLE
Fix the corrupted package.json

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12
| Bug fix?        | kinda
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Review by NoResponseMate — DISMISSED** [review]

**Review by Wojdylak — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
-    "slick-carousel": "^1.8.1"
+  "name": "@sylius-ui/frontend",
+  "version": "1.0.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Sylius/Sylius.git"
   },
-  "devDependencies": {
+  "license": "MIT",
+  "author": "Paweł Jędrzejewski",
```
Comment (diimpp) [inline package.json]: Shouldn't it be?
```suggestion
  "author": ""Sylius Sp. z o.o.",
```

**Review by diimpp — COMMENTED** [review]
@jakubtobiasz could you please add context why merging dependencies and devDependencies solves some issue?

File: package.json (hunk 1 above)
Comment (jakubtobiasz) [reply package.json]: `@sylius-ui/frontend` is a meta package. The end project requires all those deps, and `@sylius-ui/frontend` should be required in the dev deps section in a Sylius end-app.

File: package.json [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
-    "slick-carousel": "^1.8.1"
+  "name": "@sylius-ui/frontend",
+  "version": "1.0.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Sylius/Sylius.git"
   },
-  "devDependencies": {
+  "license": "MIT",
+  "author": ""Sylius Sp. z o.o.",
```
Comment (diimpp) [inline package.json]: Sorry :pray: 

```suggestion
  "author": "Sylius Sp. z o.o.",
```

**Review by diimpp — APPROVED** [review]

**Review by Wojdylak — APPROVED** [review]

**Review by Rafikooo — APPROVED** [review]

**Pull request merged**